distribution/kernel/carbon.product

```diff
@@ -13,7 +13,7 @@
    </plugins>
 
    <features>
-      <feature id="org.wso2.carbon.core.runtime" version="4.4.11"/>
+      <feature id="org.wso2.carbon.core.runtime" version="4.4.11.SNAPSHOT"/>
```

Comment:
When use the maven build plugin we need to remove the ".SNAPSHOT" part from the carbon.product. Otherwise, it will cause the build failure. Therefore this is not required.